src/Exceptions/HttpImateapot.php

```diff
@@ -8,4 +8,3 @@ class HttpImateapot extends HttpException {
     public const DESCRIPTION = "Any attempt to brew coffee with a teapot should result in the error code 418 I'm a teapot.";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/418";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces